flzoo/domainnet/domainnet_fedavg_resnet_config.py

```diff
@@ -0,0 +1,35 @@
+from easydict import EasyDict
+
+exp_args = dict(
+    data=dict(
+        dataset='domainnet',
+        data_path='./data/domainnet',
+        sample_method=dict(name='cross_domain_iid', train_num=500, test_num=30000),
+        domains='clipart,infograph,painting,quickdraw,real,sketch',
+        base_dir='D:/My_Codes/Federate-Learning/Data/DomainNet/'  
+    ),
+    learn=dict(
+        device='cuda:0', 
+        local_eps=1, 
+        global_eps=500, 
+        batch_size=64, 
+        optimizer=dict(name='sgd', lr=0.01, momentum=0.5)
+    ),
+    model=dict(
+        name='resnet18',
+        input_channel=3,
+        class_number=10,
+    ),
+    client=dict(name='cross_domain_base_client', client_num=1),
```

Comment:
Does ``client_num=1`` means the number of clients for each domain is 1? Clarify it and add more comments.